pennylane/workflow/interfaces/tensorflow.py

```diff
@@ -88,17 +88,13 @@ def vjp(*dy):
 >>> with tf.GradientTape(persistent=True) as tape:
 ...         y = f(x)
 >>> tape.jacobian(y, x)
-In the VJP function with:  (<tf.Tensor: shape=(2,), dtype=float32, numpy=array([1., 0.], dtype=float32)>,)
-eager?  True
-In the VJP function with:  (<tf.Tensor: shape=(2,), dtype=float32, numpy=array([1., 0.], dtype=float32)>,)
+In the VJP function with:  (<tf.Tensor: shape=(), dtype=float32, numpy=1.0>,)
 eager?  True
-In the VJP function with:  (<tf.Tensor: shape=(2,), dtype=float32, numpy=array([0., 1.], dtype=float32)>,)
+In the VJP function with:  (<tf.Tensor: shape=(), dtype=float32, numpy=1.0>,)
 eager?  True
-<tf.Tensor: shape=(2, 2), dtype=float32, numpy=
-array([[0.2, 0. ],
-       [0. , 0.4]], dtype=float32)>
+<tf.Tensor: shape=(), dtype=float32, numpy=0.2>
 
-As you can see, we got 3 calls to ``vjp`` instead of 2, and the first calls have identical ``dy``. We do not want
+As you can see, we got 2 calls to ``vjp`` instead of 1, and the calls have identical ``dy``. We do not want

```

Comment:
do you know why this changed?

Reply:
I think there might have been a mix-up between the example shown and the example being referred to in the text. The example is supposed to have a scalar value for the jacobian, and the text said that there should have been only one extra call to the vjp function (originally there was only 1 call so 2 should've been expected). Moreover, the output of the jacobian in the example became a 2x2 matrix instead of being scalar.